server/package.json

```diff
@@ -19,8 +19,10 @@
   "dependencies": {
     "@googlemaps/google-maps-services-js": "^3.3.16",
     "@sentry/node": "^7.28.1",
-    "adm-zip": "^0.5.9",
-    "axios": "1.2.1",
+    "@types/multer": "^1.4.7",
+    "@types/sanitize-html": "^2.8.0",
```

Comment:
Jetzt landen die Types schon wieder in den normalen Dependencies. 
Die hatte ich doch extra da raus gekehrt.